examples/json/hello/func.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/fnproject/fdk-go"
+	"os"
+)
+
+func main() {
+	fdk.Handle(fdk.HandlerFunc(myHandler))
+}
+
+func myHandler(ctx context.Context, in io.Reader, out io.Writer) {
+	var person struct {
+		Name string `json:"name"`
+	}
+	json.NewDecoder(in).Decode(&person)
+
+	if person.Name == "" {
+		person.Name = "world"
+	}
+	body := fmt.Sprintf("Hello %s!\n", person.Name)
+	err := json.NewEncoder(out).Encode(body)
```

Comment:
it would be nice if this mimicked the other example we have, with an object. this isn't really a json response, it's a string (which will encode, sure)